R/parse_params.R

```diff
@@ -37,24 +37,42 @@ parse_params <- function(station_id, start, end, hour = FALSE) {
 
   # Parse Dates -------------------------------------------------------------
 
+  if (is.null(start) & is.null(end)) {
+    message("Querying most recent day of data")
```

Comment:
```suggestion
    message("Querying data from yesterday")
```

Reply:
Just to avoid any confusion that might arise from one thinking that `most recent day` equals `most recent 24 hours`, regardless of date